app/models/user.rb

```diff
@@ -1,9 +1,16 @@
 class User < ApplicationRecord
   has_many :meetings, dependent: :destroy
   has_many :comments, through: :meetings, dependent: :destroy
+  has_many :payment_cards, dependent: :destroy
+
+  validates :phone, phone: true
 
   # Include default devise modules. Others available are:
   # :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable, :confirmable
+
+  def admin?
+    AdminUser.find_by(email: self.email).present? ? true : false
```

Comment:
AdminUser.find_by(email: self.email).present? 
this is sufficient to return true or false